tfhe/docs/how_to/debug.md

```diff
@@ -0,0 +1,66 @@
+# Debugging FHE code
```

Comment:
I believe the C needs to be capitalized as it's in a "level 1 title", but I'm not quite sure of that with the current standard of doc